test/async/mandos/promises/parent-child-async/no_async.scen.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
suggestion to use camel case instead of snake case in the filename

Reply:
Mandos scenario filenames follow a different naming convention.